doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/refreshing-data-sets-con.adoc

```diff
@@ -0,0 +1,5 @@
+[id='refreshing-data-sets-con_{context}']
+= Data refresh
+
+The data refresh feature enables you to specify an interval of time after which a data set (or data) is refreshed.
```

Comment:
I think it is worth mentioning where this feature can be found so that the user is able to use it

Reply:
I have added the location of this feature in the content.